src/user/dashboard/insights/components/memberInfo.scss

```diff
@@ -136,27 +136,27 @@
   }
 }
 .list-group {
-  width: 14vw;
+  //  width: 14vw;
 }
 .popover-body {
   width: 9vw;
   text-align: center;
   .overlay_list_btn {
     cursor: pointer;
-     background: transparent;
```

Comment:
@AuraOfDivinity Also please take care of this, I think your extension is making some unnecessary changes

Reply:
@Rupeshiya I'm not exactly sure why it's like this. Basically nothing is getting changed in this file...

Reply:
@AuraOfDivinity can you please fix this